spec/unit/type/ec2_securitygroup_spec.rb

```diff
@@ -31,4 +31,14 @@
       expect(type_class.parameters).to be_include(param)
     end
   end
+
+  it 'should order tags on output' do
```

Comment:
I wonder if this test should be moved to a shared location give its the same basic test for all resources supporting tags?